lib/lua/CMakeLists.txt

```diff
@@ -174,6 +174,8 @@ ELSE(FSO_USE_LUAJIT)
 			lzio.h
 
 			lua.natvis
+
+			bitop/bit.c
```

Comment:
This is not a standard library feature and as such must not be included in the lua code since platforms with native Lua packages will not work properly.